documentation/api/query/v4/facts.markdown

```diff
@@ -100,10 +100,10 @@ Subquery against `/fact-contents` to get all remotely-authenticated trusted fact
 Use `count` and `group_by` to get tallies of each operating system in your
 infrastructure:
 
-    curl -X GET http://localhost:8080/pdb/query/v4/facts --data-urlencode \
-      'query=["extract",[["function","count"],"value"],
-               ["=","name","operatingsystem"],
-               ["group_by", "value"]]'
+    curl -X GET http://localhost:8080/pdb/query/v4/facts \
+    --data-urlencode 'query=["extract",[["function","count"],"value"],
```

Comment:
*small niggle* the other examples include a 2 space indent on the second line, I guess it makes me feel more aware this is a multi-line example.